File: MPI/mpicheck/tests/functions/3.0/MPI_Comm_dup_with_info.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_comm_dup_with_info(var_0, var_1, var_2, var_3)
       call pmpi_comm_dup_with_info(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Comm_dup_with_info.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    MPI_Info var_1;
    MPI_Comm *var_2;
    int ret;
    /* calls */
    ret = MPI_Comm_dup_with_info(var_0, var_1, var_2);
    ret = PMPI_Comm_dup_with_info(var_0, var_1, var_2);
    return 0;
}
